From d816f79fffbcea55904bd32c1a7b7a3cb18b7bdc Mon Sep 17 00:00:00 2001
From: Song Bing <b06498@freescale.com>
Date: Thu, 18 Jul 2013 15:17:24 +0800
Subject: [PATCH 23/50] 43075638: Add time position support to API
 OMXMetadataRetr...

If input time bigger than zero, use input time to get thumbnail.

Signed-off-by: Song Bing b06498@freescale.com
---
 include/media/OMXMetadataRetriever.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/media/OMXMetadataRetriever.h b/include/media/OMXMetadataRetriever.h
index 8560e9c..cc0b02b 100644
--- a/include/media/OMXMetadataRetriever.h
+++ b/include/media/OMXMetadataRetriever.h
@@ -43,7 +43,7 @@ public:
     virtual status_t    setDataSource(int fd, int64_t offset, int64_t length);
     virtual status_t    setMode(int mode);
     virtual status_t    getMode(int* mode) const;
-    virtual VideoFrame* captureFrame();
+    virtual VideoFrame* captureFrame(int64_t timeUs, int option);
     virtual VideoFrame* getFrameAtTime(int64_t timeUs, int option);
     virtual MediaAlbumArt* extractAlbumArt();
     virtual const char* extractMetadata(int keyCode);
-- 
1.8.0

